plugins/battery/battery.plugin.zsh

```diff
@@ -110,13 +110,19 @@ elif [[ "$OSTYPE" = freebsd* ]]; then
 elif [[ "$OSTYPE" = linux*  ]]; then
 
   function battery_is_charging() {
-    ! acpi 2>/dev/null | command grep -v "rate information unavailable" | command grep -q '^Battery.*Discharging'
+    if (( $+commands[acpitool] )); then
+      ! apcitool 2>/dev/null | command grep -q '^\s+Battery.*Discharging'
+    else if (( $+commands[acpi] )); then
```

Comment:
The proper keyword is `elif`. This opens a new `if` and that's why you need the awkward `fi; fi` line below.

Reply:
Thanks, i figured there had to be one but i couldn't find it on a quick google.